src/index.ts

```diff
@@ -109,14 +112,52 @@ export const bootstrapEmpackPackedEnvironment = async (
         });
 
         sharedLibsMap[pkg.name] = getSharedLibs(extractedPackage, '');
-
+        paths[pkg.filename] = formPackagesPaths(extractedPackage, '');
```

Comment:
Since it's a simple function that's used only once, let's inline the code here.

```suggestion
        paths[pkg.filename] = {};
        Object.keys(files).forEach(filename => {
            paths[pkg.filename][filename] = `${prefix}/${filename}`;
        });
```